<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleCodeEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="handleWidth">
        <number>0</number>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="openedFiles">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Opened Files</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="styleSheet">
            <string notr="true">background-color:#f4f4f4;
border-color: rgb(244, 244, 244);
</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="fileExplorer">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>FileExplorer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="treeView">
           <property name="styleSheet">
            <string notr="true">background-color:#f4f4f4;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabShape">
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="widget" native="true"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenDirectory"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_AS"/>
    <addaction name="actionSave_All"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_Preview"/>
    <addaction name="actionNew_Window"/>
    <addaction name="actionClose_Window"/>
    <addaction name="actionSettings"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All_Files"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuIndent">
     <property name="title">
      <string>Indent</string>
     </property>
     <addaction name="actionAdd_Indent"/>
     <addaction name="actionDecrease_Indent"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionDeselect"/>
    <addaction name="menuIndent"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuShow_Panels">
     <property name="title">
      <string>Show Panels</string>
     </property>
     <addaction name="actionFile_Explorer"/>
     <addaction name="actionOpened_Files"/>
    </widget>
    <widget class="QMenu" name="menuLine_Ending">
     <property name="title">
      <string>Line Ending</string>
     </property>
     <addaction name="actionWindows"/>
     <addaction name="actionMac"/>
     <addaction name="actionUnix"/>
    </widget>
    <widget class="QMenu" name="menuSyntax">
     <property name="title">
      <string>Syntax</string>
     </property>
     <addaction name="actionNormal"/>
     <addaction name="actionBash"/>
     <addaction name="actionBatch_File"/>
     <addaction name="actionC"/>
     <addaction name="actionCpp"/>
     <addaction name="actionCSharp"/>
     <addaction name="actionCMake"/>
     <addaction name="actionCoffeeScript"/>
     <addaction name="actionCSS"/>
     <addaction name="actionD"/>
     <addaction name="actionDiff"/>
     <addaction name="actionHTML"/>
     <addaction name="actionFortan"/>
     <addaction name="actionFortan77"/>
     <addaction name="actionJava"/>
     <addaction name="actionJavaScript"/>
     <addaction name="actionJSON"/>
     <addaction name="actionLua"/>
     <addaction name="actionMakeFile"/>
     <addaction name="actionMarkDown"/>
     <addaction name="actionMatLab"/>
     <addaction name="actionPascal"/>
     <addaction name="actionProperties"/>
     <addaction name="actionPerl"/>
     <addaction name="actionPython"/>
     <addaction name="actionRuby"/>
     <addaction name="actionYAML"/>
     <addaction name="actionTeX"/>
     <addaction name="actionSQL"/>
     <addaction name="actionXML"/>
    </widget>
    <addaction name="menuShow_Panels"/>
    <addaction name="menuLine_Ending"/>
    <addaction name="menuSyntax"/>
    <addaction name="actionWordWrap"/>
    <addaction name="actionShow_Linenumbers"/>
    <addaction name="actionToolBar"/>
   </widget>
   <widget class="QMenu" name="menuFind">
    <property name="title">
     <string>Find</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionFind_Next"/>
    <addaction name="actionFind_Prev"/>
    <addaction name="actionReplace"/>
    <addaction name="actionReplace_All"/>
   </widget>
   <widget class="QMenu" name="menuPrefrences">
    <property name="title">
     <string>Prefrences</string>
    </property>
    <widget class="QMenu" name="menuThemes">
     <property name="title">
      <string>Themes</string>
     </property>
     <addaction name="actionSolarizedDark"/>
    </widget>
    <addaction name="menuThemes"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFind"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
   <addaction name="menuPrefrences"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_AS"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_AS">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="window-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpenDirectory">
   <property name="text">
    <string>Open Directory</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="document-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionDeselect">
   <property name="text">
    <string>Deselect</string>
   </property>
  </action>
  <action name="actionFile_Explorer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>File Explorer</string>
   </property>
  </action>
  <action name="actionOpened_Files">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Opened Files</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="icon">
    <iconset theme="window-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionClose_Window">
   <property name="icon">
    <iconset theme="window-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Close Window</string>
   </property>
  </action>
  <action name="actionUnix">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Unix</string>
   </property>
  </action>
  <action name="actionWindows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Windows</string>
   </property>
  </action>
  <action name="actionMac">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mac</string>
   </property>
  </action>
  <action name="actionClose_All_Files">
   <property name="text">
    <string>Close All Files</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="text">
    <string>Find Next</string>
   </property>
  </action>
  <action name="actionFind_Prev">
   <property name="text">
    <string>Find Prev</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionReplace_All">
   <property name="text">
    <string>Replace All</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ToolBar</string>
   </property>
  </action>
  <action name="actionJSON">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>JSO&amp;N</string>
   </property>
  </action>
  <action name="actionBatch_File">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Batch F&amp;ile</string>
   </property>
  </action>
  <action name="actionCoffeeScript">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C&amp;offeeScript</string>
   </property>
  </action>
  <action name="actionCSharp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C#</string>
   </property>
  </action>
  <action name="actionJava">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Java</string>
   </property>
  </action>
  <action name="actionCSS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CSS</string>
   </property>
  </action>
  <action name="actionJavaScript">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ja&amp;vaScript</string>
   </property>
  </action>
  <action name="actionMakeFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ma&amp;keFile</string>
   </property>
  </action>
  <action name="actionMatLab">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MatLab</string>
   </property>
  </action>
  <action name="actionPascal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Pascal</string>
   </property>
  </action>
  <action name="actionPython">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>P&amp;ython</string>
   </property>
  </action>
  <action name="actionBash">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Bash</string>
   </property>
  </action>
  <action name="actionCMake">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CMake</string>
   </property>
  </action>
  <action name="actionCpp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C++</string>
   </property>
  </action>
  <action name="actionFortan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Fortan</string>
   </property>
  </action>
  <action name="actionXML">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;XML</string>
   </property>
  </action>
  <action name="actionLua">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Lua</string>
   </property>
  </action>
  <action name="actionMarkDown">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;MarkDown</string>
   </property>
  </action>
  <action name="actionPerl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>P&amp;erl</string>
   </property>
  </action>
  <action name="actionRuby">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Ruby</string>
   </property>
  </action>
  <action name="actionSQL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;SQL</string>
   </property>
  </action>
  <action name="actionTeX">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;TeX</string>
   </property>
  </action>
  <action name="actionC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </action>
  <action name="actionYAML">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>YAML</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionDiff">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Diff</string>
   </property>
  </action>
  <action name="actionFortan77">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fortan&amp;77</string>
   </property>
  </action>
  <action name="actionD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;D</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionShow_Linenumbers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Linenumbers</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="actionStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>StatusBar</string>
   </property>
  </action>
  <action name="actionWordWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>WordWrap</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionAdd_Indent">
   <property name="text">
    <string>Add Indent</string>
   </property>
  </action>
  <action name="actionDecrease_Indent">
   <property name="text">
    <string>Decrease Indent</string>
   </property>
  </action>
  <action name="actionPrefrences">
   <property name="text">
    <string>Prefrences</string>
   </property>
  </action>
  <action name="actionSolarizedDark">
   <property name="text">
    <string>SolarizedDark</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
